client/js/lib/prosemirror/converter.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * This ProseMirrorConverter class is responsible for converting
+ * ProseMirror data to an HTML string and to convert proseMirror data
+ * to an HTML string.
```

Comment:
Nitpick: something seems to be duplicated here.

Reply:
Ups, thanks a lot, this was adjusted here, too https://github.com/demos-europe/demosplan-core/pull/3875/commits/4ba7fe8b305eb22aa506d0e9ae95102779fa1faa